terraform/modules/monitoring/alert.tf

```diff
@@ -0,0 +1,132 @@
+resource "google_monitoring_notification_channel" "basic" {
+  project = var.project_id
+
+  display_name = "owner notification channel"
+
+  type = "email"
+
+  labels = {
+    email_address = var.notification_channel_email
+  }
+
+  force_delete = false
+}
+
+resource "google_monitoring_alert_policy" "public_key_service_alert" {
+  project = var.project_id
+
+  display_name = "Public-Key Service Alert: Too many none-200 response"
+
+  combiner = "AND"
+
+  conditions {
+    display_name = "Too many None-200 response"
+
+    condition_threshold {
+      filter          = "metric.type=\"run.googleapis.com/request_count\" resource.type=\"cloud_run_revision\" resource.label.\"service_name\"=\"${var.public_key_service_name}\" AND metric.label.\"response_code\"!=\"200\""
+      duration        = "0s"
```

Comment:
Does this make sense?

Reply:
Yes, my point is at any time, if the request count exceed the threshold, we send out alert.